src/base/Field.php

```diff
@@ -43,7 +43,7 @@
  * @author Pixel & Tonic, Inc. <support@pixelandtonic.com>
  * @since 3.0.0
  */
-abstract class Field extends SavableComponent implements FieldInterface, Iconic, Actionable
+abstract class Field extends SavableComponent implements FieldInterface, Iconic, Actionable, \Stringable
```

Comment:
What is the point of adding this? Just to be explicit? Seems unnecessary. At the very least, it should be imported so we don’t get that ugly `\`.

Reply:
@brandonkelly I imported Stringable to use statement https://github.com/craftcms/cms/pull/16711/commits/0ef22bea0d202a8ee6ec97d63443155669314c24 👍